nginx.conf

```diff
@@ -0,0 +1,59 @@
+# User and worker settings
+user www-data;
+worker_processes auto;
+pid /run/nginx.pid;
+include /etc/nginx/modules-enabled/*.conf;
+
+# Events block for handling connections
+events {
+    worker_connections 768;
+    # multi_accept on;
+}
+
+# HTTP block for handling HTTP requests
+http {
+    # Basic Settings
+    sendfile on;
+    tcp_nopush on;
+    tcp_nodelay on;
+    keepalive_timeout 65;
+    types_hash_max_size 2048;
+    server_tokens off;
+
+    # Include MIME types
+    include /etc/nginx/mime.types;
+    default_type application/octet-stream;
+
+    # SSL Settings
+    ssl_protocols TLSv1 TLSv1.1 TLSv1.2; # Dropping SSLv3, ref: POODLE
+    ssl_prefer_server_ciphers on;
+
+    # Logging Settings
+    access_log /var/log/nginx/access.log;
+    error_log /var/log/nginx/error.log;
+
+    # Gzip Settings
+    gzip on;
+    gzip_disable "msie6";
+
+    # Virtual Host Configs
+    include /etc/nginx/conf.d/*.conf;
+    include /etc/nginx/sites-enabled/*;
+
+    # Reverse proxy configuration
+    server {
+        listen 80;
+        # Server name can be domain or IP
+        server_name idrinth-api-ben.ch;
```

Comment:
this server will never be called, we are looking for localhost here - you are not leaving the machine.

Reply:
what can we do here? use 'http://localhost:8080'?

Reply:
you can stay on port 80, the server_name just has to be something you can actually use, 127.0.0.1 or localhost for example